trello_service.py

```diff
@@ -0,0 +1,107 @@
+import requests
+import configparser
+import board as b
+import logging
+
+config = configparser.ConfigParser()
+config.read('.cfg')
+trello = config['TRELLO']
+
+
+def get_cards():
+    all_cards_url = 'https://api.trello.com/1/boards/{0}/cards'.format(trello['BOARD_ID'])
```

Comment:
Minor point, but it's usually nicer to write format strings in the form: `f'https://api.trello.com/1/boards/{trello['BOARD_ID']}/cards'`